tests/fixtures.py

```diff
@@ -30,6 +30,15 @@ def load_config():
         raise Exception("KUBECONFIG file not found: {0}".format(kube_config_path))
 
 
+@pytest.fixture(scope='session')
+def set_docker_images():
+    docker_image = os.getenv('CLIENT_DOCKER_IMAGE', '')
+    if docker_image:
+        print("Set docker images to: {0}".format(docker_image))
```

Comment:
where is this printed? maybe we should store it in a log file in case we cant see the print?

Reply:
Currently we don't have log for tests other than stdout